plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/detection_output.cpp

```diff
@@ -0,0 +1,91 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "single_layer_tests/detection_output.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+
+const int numClasses = 5;
+const int backgroundLabelId = 0;
+const std::vector<int> topK = {3};
+// const std::vector<std::vector<int>> keepTopK = { {50}, {100} };
+const std::vector<std::vector<int>> keepTopK = {{9}, {10}, {11}};
+const std::vector<std::string> codeType = {"caffe.PriorBoxParameter.CORNER", "caffe.PriorBoxParameter.CENTER_SIZE"};
+// const std::vector<std::string> codeType = {"caffe.PriorBoxParameter.CORNER"};
```

Comment:
Is this line redundant?

Reply:
Only the source code is cleaned up and the test is not yet.

Reply:
The test cases is not determined and may be changing quickly so I temporarily leave it as draft.